### ISSUE TITLE
Certain page elements do no update when refreshing the page

### ISSUE THREAD

**Issue by bernep** [bnidevs/the-hotbox#64]
Not all of the parameters on the page refresh to their default state when the page is refreshed. (e.g. the sliders will not move when refreshing the page, upload button doesn't empty the video file when refreshing the page).

**Comment by bernep** [comment]
There is now a setDefaults9) function in index.js. Perhaps someone can add an event handler or something for refreshing the page that calls setDefaults()

**Comment by bernep** [comment]
Something else I noticed. If you upload a video, and refresh the page, the upload bar will still be there as well when it shouldn't be. I imagine this will probably happen with the processing bar and the preview image.

Overall this isn't a huge issue, but yeh.

**Comment by bernep** [comment]
Slider values fixed, but stuff like the upload button (and potentially any sort of progress bar or image previewing) will not refresh on a page refresh.
EDIT: This seems to only happen on Firefox (and it may be just my Firefox installation). Chrome doesn't seem to have any refresh issues.

**Comment by MilkTeaBBT** [comment]
I looked around and found refreshing issue is just a problem with how Firefox does caching. Firefox preserves the cache to make navigation faster and only clears it when you close the browser. The fastest solution is just to refresh by hitting `ctrl` + `F5` or hitting `shift` when you hit the refresh button. I found a page describing possible solutions that I'll include [here](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/1.5/Using_Firefox_1.5_caching) but I'm honestly not sure if anything really needs to be done about this.
